.github/workflows/minecraft-crash.yml

```diff
@@ -48,6 +48,10 @@ jobs:
             const issueTitle = issueData.title
             const issueBody = issueData.body
 
+            // Uses negative lookbehind and lookahead to match start and end of string as well
+            const minecraftRegex = /(?<![a-z])Minecraft(?![a-z])/i
```

Comment:
Have adjusted the regex pattern slightly by replacing `\w` with `[a-z]` this way it also detects strings such as "_minecraft". Checking for uppercase `A-Z` is not necessary because the pattern is case insensitive (`/i`).